### ISSUE TITLE
webvision训练代码中疑似一些逻辑错误

### ISSUE THREAD

**Issue by wangjie-123** [lijichang/LNL-NCE#6]
1、干净样本的选择再次筛选逻辑好像永远干净的样本都是空集合
    prob = np.array(sver_collection)
    # prob = 1.0 - np.array(sver_collection)
    mask_lab = prob < args.threshold_sver
    mask_unl = prob > args.threshold_sver

    labeledFeatures = trainFeatures[mask_lab]
    labeledLogits = trainLogits[mask_lab]
    labeledNoisyLabels = trainNoisyLabels[mask_lab]
    labeledW = prob[mask_lab]

    knn_labeledLogits = labeledLogits[labeledW > 0.95]
    knn_labeledFeatures = labeledFeatures[labeledW > 0.95]
    knn_labeledNoisyLabels = labeledNoisyLabels[labeledW > 0.95]
    
    上述threshold_sver为0.9，这样knn_labeledLogits永远都是空集合。labeledW 应该是prob = 1.0 - prob[mask_lab]才对吧。
    
   2、test测试代码并没有top1和top5的概念。
   def test(net1, net2, test_loader):
    net1.eval()
    net2.eval()

    correct = 0
    total = 0

    with torch.no_grad():
        for batch_idx, (inputs, targets) in enumerate(test_loader):
            inputs, targets = inputs.cuda(), targets.cuda()
            outputs1 = net1(inputs)
            outputs2 = net2(inputs)
            outputs = outputs1 + outputs2

            _, predicted = torch.max(outputs, 1)
            _, predicted1 = torch.max(outputs1, 1)
            _, predicted2 = torch.max(outputs2, 1)
            
            correct += predicted.eq(targets).cpu().sum().item()
            correct1 += predicted1.eq(targets).cpu().sum().item()
            correct2 += predicted2.eq(targets).cpu().sum().item()
            
            total += targets.size(0)

    acc = 100. * correct / total
    acc1 = 100. * correct1 / total
    acc2 = 100. * correct2 / total

    return acc, acc1, acc2
    
    这段代码中的correct 只能是top1吧，并没有计算top5
    
    3、应该是push代码是推的修改版本，麻烦您抽空可以更新下正确代码，谢谢您了，您的方法对我很有启发~

**Comment by wangjie-123** [comment]
# mixup regularization for labeled data
        Lx = -torch.mean(torch.sum(F.log_softmax(mixed_logits, dim=1) * mixed_target, dim=1))

        # penalty regularization for mixed labeled data
        prior = torch.ones(args.num_class) / args.num_class
        prior = prior.cuda()
        pred_mean = torch.softmax(mixed_logits, dim=1).mean(0)
        penalty = torch.sum(prior * torch.log(prior / pred_mean))

        # overall loss
        loss = Lx + penalty
        
        请教下webvision训练中没有使用修正后的标签吗？好像这个loss只有Lx和penalty？
